src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -1,8 +1,27 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.Ports;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 public class Intake extends SubsystemBase {
-    // TODO: Create empty intake methods (e.g. move up, move down, run rollers, etc.)
-    // TODO: start implementing methods
+    
+    private final CANSparkMax m_rollerMotor = new CANSparkMax(Ports.ROLLER_MOTOR_PORT, MotorType.kBrushless); //may be actually brushed not sure rn
+    private final CANSparkMax m_intakeMechanismMotor = new CANSparkMax(Ports.INTAKE_MECHANISM_MOTOR_PORT, MotorType.kBrushless); //^^f
+
+    public void moveUp(final int degrees) { 
```

Comment:
Generally, subsystems should contain stuff you can tell the motor controllers in one iteration. They should be limited in the amount of state they store.

Therefore these three methods should probably take a speed (between -1 and 1) not a distance to move.